config.default

```diff
@@ -59,8 +59,8 @@ NAMESPACE=gcr.io
 
 # Upstream image
 BASE_IMAGE=php
-BASE_NAMESPACE=circleci
-BASE_TAG=7.3-node
+BASE_NAMESPACE=cimg
```

Comment:
right, circleci has python2 on Debian 10, cimg has python3 on Ubuntu 20.04; cimg seems to be part of "Next gen service images" https://circleci.com/docs/2.0/circleci-images/#next-gen-service-images

Reply:
which sounds like a good choice https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/

Reply:
Indeed, I'm having a shot at making it work :crossed_fingers: